CHANGELOG.asciidoc

```diff
@@ -3,30 +3,152 @@
 :issue: https://github.com/elastic/beats/issues/
 :pull: https://github.com/elastic/beats/pull/
 
-[[release-notes-8.0.0-rc2]]
-=== Beats version 8.0.0-rc2
+[[release-notes-8.0.0]]
+=== Beats version 8.0.0
+https://github.com/elastic/beats/compare/v7.17.0...v8.0.0[View commits]
 
-Changes will be described in the GA release.
+==== Breaking changes
+
+*Affecting all Beats*
+
+- `include_matches` option of `journald` input no longer accepts a list of string. {pull}29294[29294]
+- `add_fields` processor is now able to set metadata in events {pull}30092[30092]
+
+*Auditbeat*
+
+
+*Filebeat*
+
+- Add `while_pattern` type to multiline reader. {pull}19662[19662]
+- auditd dataset: Use process.args to store program arguments instead of auditd.log.aNNN fields. {pull}29601[29601]
+- Remove deprecated old awscloudwatch input name. {pull}29844[29844]
+
+*Heartbeat*
+
+- Add fonts to support more different types of characters for multiple languages. {pull}29861[29861]
```

Comment:
I wouldn't consider this a breaking change (cc @andrewvc / @lucasfcosta / @vigneshshanmugam)

Reply:
already discussed in another PR, I will fix it here as well, thanks!

Reply:
actually it was a duplicate 🤦‍♂️ , fixed